### PR TITLE
Avoid "select *" query

### PR DESCRIPTION
**Pull request by lt5227**
Simple optimization of trace sql query statement. Avoid "select *" query method

- [x] Update the [CHANGES log.](https://github.com/apache/skywalking/blob/master/CHANGES.md)

### CODE REVIEW

**Comment by wu-sheng** [comment]
Thanks for submitting. Please update the changes.md kn the root folder.

File: oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java [hunk 1]
```diff
@@ -177,7 +177,7 @@ public TraceBrief queryBasicTraces(long startSecondTB,
             buildLimit(sql, from, limit);
 
             try (ResultSet resultSet = h2Client.executeQuery(
-                connection, "select * " + sql.toString(), parameters.toArray(new Object[0]))) {
+                connection, "select segment_id, start_time, endpoint_name, latency, is_error, trace_id " + sql.toString(), parameters.toArray(new Object[0]))) {
```
Comment (wu-sheng) [inline oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:180]: Are these all fields we define in the query protocol?

**Comment by wu-sheng** [comment]
Please check your code style. We have provided the control file jlin the root for IntelliJ. And you could check through mvn check.

**Comment by wu-sheng** [comment]
> 2021-07-26T10:27:50.5906240Z Audit done.
2021-07-26T10:27:50.5907589Z [INFO] There are 6 errors reported by Checkstyle 8.19 with /home/runner/work/skywalking/skywalking/apm-checkstyle/checkStyle.xml ruleset.
2021-07-26T10:27:50.5916205Z [ERROR] src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:[181,60] (whitespace) WhitespaceAround: '+' is not preceded with whitespace.
2021-07-26T10:27:50.5919035Z [ERROR] src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:[182,60] (whitespace) WhitespaceAround: '+' is not preceded with whitespace.
2021-07-26T10:27:50.5921575Z [ERROR] src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:[183,63] (whitespace) WhitespaceAround: '+' is not preceded with whitespace.
2021-07-26T10:27:50.5923945Z [ERROR] src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:[184,57] (whitespace) WhitespaceAround: '+' is not preceded with whitespace.
2021-07-26T10:27:50.5926334Z [ERROR] src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:[185,58] (whitespace) WhitespaceAround: '+' is not preceded with whitespace.
2021-07-26T10:27:50.5930350Z [ERROR] src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:[223,53] (whitespace) WhitespaceAround: '+' is not followed by whitespace.

**Review by lt5227 — COMMENTED** [review]
> Please check your code style. We have provided the control file jlin the root for IntelliJ. And you could check through mvn check.



> > 2021-07-26T10:27:50.5906240Z Audit done.
> > 2021-07-26T10:27:50.5907589Z [INFO] There are 6 errors reported by Checkstyle 8.19 with /home/runner/work/skywalking/skywalking/apm-checkstyle/checkStyle.xml ruleset.
> > 2021-07-26T10:27:50.5916205Z [ERROR] src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:[181,60] (whitespace) WhitespaceAround: '+' is not preceded with whitespace.
> > 2021-07-26T10:27:50.5919035Z [ERROR] src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:[182,60] (whitespace) WhitespaceAround: '+' is not preceded with whitespace.
> > 2021-07-26T10:27:50.5921575Z [ERROR] src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:[183,63] (whitespace) WhitespaceAround: '+' is not preceded with whitespace.
> > 2021-07-26T10:27:50.5923945Z [ERROR] src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:[184,57] (whitespace) WhitespaceAround: '+' is not preceded with whitespace.
> > 2021-07-26T10:27:50.5926334Z [ERROR] src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:[185,58] (whitespace) WhitespaceAround: '+' is not preceded with whitespace.
> > 2021-07-26T10:27:50.5930350Z [ERROR] src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TraceQueryDAO.java:[223,53] (whitespace) WhitespaceAround: '+' is not followed by whitespace.

has been corected

**Review by wu-sheng — APPROVED** [review]
LGTM

**Pull request merged**